index.js

```diff
@@ -22,6 +22,13 @@ var pkgUp = require('pkg-up')
 var testExclude = require('test-exclude')
 var yargs = require('yargs')
 
+var ProcessInfo
+try {
+  ProcessInfo = require('./lib/process.covered.js')
```

Comment:
do we need to add `process.js` to the list of files in `./build-self-covered.js`, and should it be added to our exclude rules in the packages.json, I noticed `index.covered.js` is.